pkg/iosrc/uri_windows.go

```diff
@@ -0,0 +1,32 @@
+package iosrc
+
+import (
+	"path/filepath"
+	"regexp"
+)
+
+var winVolumeRe = regexp.MustCompile("^[a-zA-Z]:")
+
+func normalizeFilepaths(path string) (string, error) {
+	if !winVolumeRe.MatchString(path) {
+		if scheme, err := getscheme(path); err != nil || scheme != "" {
+			return path, err
```

Comment:
This will leave relative paths on windows as relative, is that right & intended?

Reply:
No, this just makes sure that already scheme-ified paths will not get mutated.